Source/WebCore/rendering/style/PositionArea.cpp

```diff
@@ -164,40 +164,54 @@ ItemPosition PositionArea::defaultAlignmentForAxis(BoxAxis physicalAxis, Writing
     return containerWritingMode.isBlockFlipped() ? flip(alignment) : alignment;
 }
 
-WTF::TextStream& operator<<(WTF::TextStream& ts, const PositionAreaSpan& span)
+WTF::TextStream& operator<<(WTF::TextStream& ts, PositionAreaAxis axis)
 {
-    ts << "{ axis: "_s;
-    switch (span.axis()) {
+    switch (axis) {
     case PositionAreaAxis::Horizontal: ts << "horizontal"_s; break;
     case PositionAreaAxis::Vertical:   ts << "vertical"_s; break;
     case PositionAreaAxis::X:          ts << 'x'; break;
     case PositionAreaAxis::Y:          ts << 'y'; break;
     case PositionAreaAxis::Block:      ts << "block"_s; break;
     case PositionAreaAxis::Inline:     ts << "inline"_s; break;
+    default:                           ASSERT_NOT_REACHED(); break;
```

Comment:
Do we really need these? Lets not add random asserts to debug code.

Reply:
I've remove those.